### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by brytebee**
In this project, I will build a simple HTML list of To Do tasks. The list will be styled according to the specifications listed by the client (Microverse requirements). This simple web page will be built using webpack, vanilla Javascript DOM manipulation and served by a webpack dev server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+## Live Demo
+
+
+
+
+### Prerequisites
+
+Basic knowledge of HTML Semantics CSS And JS-DOM to understand code.
+Ordinary internet browser knowledge to use. :smiley:
+
```
Comment (Davidosky007) [inline README.md:23]: * Kindly add a descriptive README file to your project it should contain the basic commands and instructions to successfully run your project locally. please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @brytebee 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (hemantso) [reply README.md:23]: - [ ] You made a good effort to update the README file as suggested by the reviewer. But still, there is some information left to run the project locally. Like clone the project, installing the packages, `npm run build`, `npm start` to run the project locally. Kindly try to add this instruction under the `getting started` heading.

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @brytebee, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

Alt Text
 ## Required Changes ♻️
_Check the comments under the previous reviewer's comments._

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**